### PR TITLE
Add style command, send Grass style to Nevada

### PR DESCRIPTION
**Pull request by joknelid**

### CODE REVIEW

**Comment by oskarhane** [comment]
I've added you to the contributor whitelist now so this should now build successfully.

**Comment by oskarhane** [comment]
Passed the whitelist, but failed on linter now 😄

**Comment by joknelid** [comment]
@oskarhane Sorry about that, I added "test" and "expect" to the globals for standard in package.js. Not sure why it doesn't fail on the other test files though, I might be missing something?

File: src/shared/modules/visualization/visualizationDuck.js [hunk 1]
```diff
@@ -38,11 +38,13 @@ function updateLabelData (state, labels) {
 }
 
 function updateStyleData (state, styleData) {
-  return Object.assign({}, state, {styleData: styleData})
+  const clonedStyle = Object.assign({}, styleData)
```
Comment (joknelid) [inline src/shared/modules/visualization/visualizationDuck.js]: Oskar: This is what I was trying to talk about at standup. Do I need to do a deep copy to keep the redux state immutable immutable? This works for now in practice but you wont be able to do fancy stuff like rewind/replay.

Comment (oskarhane) [inline src/shared/modules/visualization/visualizationDuck.js]: I think you can do it and simplifying the updateStyleData function at the same time.

```
const updateStyleData = (state, styleData) => ({...state, styleData: {...styleData}})
```

**Comment by oskarhane** [comment]
Change of plans. Closing this.

**Pull request closed without merging**